tasks/task406_mickey_fr_sentence_perturbation_generation.json

```diff
@@ -6,7 +6,11 @@
         "Common Sense Beyond English: Evaluating and Improving Multilingual Language Models for Commonsense Reasoning; Paper Link: https://arxiv.org/pdf/2106.06937.pdf"
     ],
     "Categories": [
-        "Text Modification"
+        "Text Modification -> Paraphrase Generation"
```

Comment:
I don't think it's paraphrasing. In this task, the new sentence can have a different meaning. Maybe we need a new category. (e.g., `Text Modification -> Perturbation`)